### PR TITLE
Docbuild automation

### PR DESCRIPTION
**Pull request by xBalbinus**
Closes: [#70](https://github.com/osmosis-labs/docs/issues/70)

## What is the purpose of the change

Slight changes in specs documentation to support the automated docbuild processes for the official docs repo.

## Brief Changelog

Added minor changes in each of the specs folders' readme in the modules (x) folder.

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / **no**)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / **no**)
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  **[Osmosis docs repo](https://github.com/osmosis-labs/docs)**   /   not documented)

### CODE REVIEW

**Review by daniel-farina — APPROVED** [review]
Some conflicts it seems.

**Comment by daniel-farina** [comment]
@xBalbinus 
There are conflicts, did you pull main before making changes?

**Comment by daniel-farina** [comment]
Your branch was quite behind:
image


Please do a new PR with the latest changes form main, it might be easier.

**Comment by xBalbinus** [comment]
I pulled from main just now, seems like the merge conflicts were actually just the changes I needed to make in the first place in the automation process for docbuild so I went through to manually pick out the changes. Wasn't much trouble.

@daniel-farina please confirm that this is good and we can merge!

File: x/gamm/spec/05_queries_transactions.md [hunk 1]
```diff
@@ -75,106 +63,77 @@ osmosisd query gamm pool <poolID> [flags]
...
-
+#### Example
 ```sh
 osmosisd query gamm pool-params 1
 ```
 
-## Pools
 
+### Pools
 Query parameters and assets of all active pools.
```
Comment (daniel-farina) [inline x/gamm/spec/05_queries_transactions.md:103]: It looks like this is being deleted, we should keep it.

**Review by daniel-farina — APPROVED** [review]
Ok I reviewed the changes. This seems good.
We iterate from here.

**Comment by ValarDragon** [comment]
Can we disable the `MD024` lint?

The others seem reasonable to fix

**Comment by daniel-farina** [comment]
@xBalbinus 
Could you fix the linting issues please
https://github.com/osmosis-labs/osmosis/runs/6513987858?check_suite_focus=true

**Comment by xBalbinus** [comment]
Yep! Fixing asap!

**Comment by daniel-farina** [comment]
> Can we disable the `MD024` lint?
> 
> The others seem reasonable to fix

I agree with alos disabling it @xBalbinus

**Pull request merged**